static/technology/chainsimulator.drawio

```diff
@@ -0,0 +1,58 @@
+<mxfile host="app.diagrams.net" modified="2024-02-27T10:39:15.564Z" agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36" etag="ph4Y5ugfU1mbP7bKnenz" version="23.1.7" type="device">
```

Comment:
If possible, perhaps use mermaid diagrams, for consistency (it's used in some places in the docs).

Reply:
hmm also other diagrams are generated from draw.io